TSEconomy/TSEconomy.cs

```diff
@@ -33,18 +36,20 @@ public override void Initialize()
             // init db
             DB.InitializeDB(Config.UseMySQL);
 
+        }
+
+        public void OnInitialize(EventArgs args)
+        {
+            Localization.SetupLanguage();
+
             // register commands
             Commands.Commands.RegisterAll();
 
             // register hooks
             TShockAPI.Hooks.GeneralHooks.ReloadEvent += (x) => {
                 Configuration.Configuration.Load();
-                x.Player.SendSuccessMessage("[TSEconomy] Reloaded config.");
+                x.Player.SendSuccessMessage(Localization.TryGetString("[i:855]Reloaded config.", "plugin"));
```

Comment:
make localization keys more simplistic? instead of attaching full string we could do something like "reloadConfig", which has a default value instead if a value hasnt been found in existing lang?

Reply:
Hmmm, we would need to add another value to the ValueKey class, change every string again to use the key instead of the string, set the default value as the string in english reserialize all of that and port the translation we currently have for spanish / english to the new xml files

Reply:
> Hmmm, we would need to add another value to the ValueKey class, change every string again to use the key instead of the string, set the default value as the string in english reserialize all of that and port the translation we currently have for spanish / english to the new xml files.
> 
> would be possible but is it worth it for the work its required?

idk just seems a little janky but i suppose if it works, it works

Reply:
yeah, tho ig it prevents stuff like having to lookup / translate the same messages since the default value is the key if theres two of the same keys an error occurs